lib/combineNodes.js

```diff
@@ -0,0 +1,367 @@
+'use strict';
+var Cesium = require('cesium');
+var jp = require('jsonpath');
+var _ = require('underscore');
```

Comment:
It does have a deep comparison `_.equals` which is really what I'm using it for. We have rewritten that function so many times in this library, I figured it might be best to try to consolidate it to a third party utility. 

@mramato I'm fine with using a different node module for deep equals, rolling our own or just keeping underscore, whatever you think is best practice here.